tests_pytest/_test_util/graph_builder_utils.py

```diff
@@ -87,7 +87,6 @@ def build_nbits_qc(a_nbits=8, a_enable=True, w_attr=None, pos_attr=(32, False, (
     Returns:
 
     """
-    assert not(a_enable and q_preserving)
```

Comment:
Don't remove the assertion.
This is a utility method that is used in other tests as well, and by removing this assertion, you're changing those tests.

If you need to use the `build_nbits_qc` functions in your tests, and this implementation is unsuitable for you, you can implement it as you wish in your test file.